packages/design-picker/src/components/design-picker-category-filter/style.scss

```diff
@@ -5,6 +5,20 @@
 $design-picker-category-filter-text-color: #3c434a;
 $design-picker-category-filter-text-color-active: var(--studio-gray-100);
 
+.design-picker {
+	&.design-picker__unified {
+		.responsive-toolbar-group__dropdown {
+			&.design-picker__category-filter {
```

Comment:
As noted previously, if we feel that the `design-picker__category-filter` class should be responsibility of the `UnifiedDesignPickerCategoryFilter` component, then we should add the class in that component:

```diff
diff --git a/packages/design-picker/src/components/design-picker-category-filter/unified-design-picker-category-filter.tsx b/packages/design-picker/src/components/design-picker-category-filter/unified-design-picker-category-filter.tsx
index e409f6a815..29ab49171b 100644
--- a/packages/design-picker/src/components/design-picker-category-filter/unified-design-picker-category-filter.tsx
+++ b/packages/design-picker/src/components/design-picker-category-filter/unified-design-picker-category-filter.tsx
@@ -4,18 +4,12 @@ import type { Category } from '../../types';
 import './style.scss';
 
 interface Props {
-	className: string;
 	categories: Category[];
 	selectedSlug: string | null;
 	onSelect: ( selectedSlug: string | null ) => void;
 }
 
-export function UnifiedDesignPickerCategoryFilter( {
-	className,
-	categories,
-	onSelect,
-	selectedSlug,
-}: Props ) {
+export function UnifiedDesignPickerCategoryFilter( { categories, onSelect, selectedSlug }: Props ) {
 	const onClick = ( index: number ) => {
 		const category = categories[ index ];
 
@@ -28,7 +22,7 @@ export function UnifiedDesignPickerCategoryFilter( {
 	const initialActiveIndex = categories.findIndex( ( { slug } ) => slug === selectedSlug );
 	return (
 		<ResponsiveToolbarGroup
-			className={ className }
+			className="design-picker__category-filter"
 			initialActiveIndex={ initialActiveIndex !== -1 ? initialActiveIndex : 0 }
 			onClick={ onClick }
 		>
diff --git a/packages/design-picker/src/components/unified-design-picker.tsx b/packages/design-picker/src/components/unified-design-picker.tsx
index 463b115501..ad11dd52cf 100644
--- a/packages/design-picker/src/components/unified-design-picker.tsx
+++ b/packages/design-picker/src/components/unified-design-picker.tsx
@@ -243,7 +243,6 @@ const DesignPicker: React.FC< DesignPickerProps > = ( {
 			<div className="design-picker__filters">
 				{ categorization && hasCategories && (
 					<UnifiedDesignPickerCategoryFilter
-						className="design-picker__category-filter"
 						categories={ categorization.categories }
 						onSelect={ categorization.onSelect }
 						selectedSlug={ categorization.selection }

```